index.d.ts

```diff
@@ -47,6 +47,8 @@ declare module "bedrock-protocol" {
     realms?: RealmsOptions
     // the path to store authentication caches, defaults to .minecraft
     profilesFolder?: string | false
+    // Called when microsoft authorization is needed when not provided it will the information log to the console instead
+    onMsaCode?: (data: MicrosoftDeviceAuthorizationResponse) => void;
```

Comment:
Could we import the types from prismarine-auth (ServerDeviceCodeResponse) in case they're changed in the future

Reply:
Yes, I didn't realise it was being exported. I Just noticed it being declared the same way in node-minecraft-protocol.

I just pushed a commit changing it.